app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,133 @@
+      GitHubRepoMapping[GitHub repository 1]
+    end
+
+    %% Define the Inventory Service Catalog Service subgraph inside Konnect Service Catalog
+    subgraph InventoryService[Inventory Service]
+      direction LR
+      SomeOtherService[Gateway service 2]
+      AnotherService[PagerDuty service 2]
+      DifferentRepo[GitHub repository 2]
+    end
+
+  end
+
+  %% Connect the resources to the Konnect Service Catalog
+  GitHub -- "GitHub repositories" --> KonnectServiceCatalog
+  PagerDuty -- "PagerDuty services" --> KonnectServiceCatalog
+  GatewayManager -- "Gateway services" --> KonnectServiceCatalog
+
+  %% Style the subgraphs
+  class KonnectServiceCatalog konnect
+  class BillingService serviceCatalog
+  class InventoryService serviceCatalog
+{% endmermaid %}
+<!-- vale on-->
+
+> Figure 1: This diagram shows how you can use both external integrations, like GitHub and PagerDuty, as well as built-in integrations like Gateway Manager to pull resources into Service Catalog. You can then map those resources (like GitHub repositories, PagerDuty services, and Gateway services) to Service Catalog Services.
+
+## Service Catalog use cases
+
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Keep track of your organization's service ownership by mapping teams to Services. | Add the Service's owner when you [create a new Service in {{site.konnect_short_name}}](https://cloud.konghq.com/service-catalog/create-service) |
+| Gain visibility into all your Services, including unrecognized or undiscovered APIs in your organization.  | [Service Catalog integrations](https://cloud.konghq.com/service-catalog/integrations) |
+| Consolidate key Service health metrics, documentation, and API specs into a single dashboard, allowing you to interact with other tools from one place. | [Service Catalog dashboard](https://cloud.konghq.com/service-catalog/) |
+
+<!-- commenting this out until it's released:
+| Govern how services are created and maintained across your company to adhere to security, compliance, and engineering best practices. | Scorecards |-->
+
+## Service Catalog terminology
+
+| Term | Definition |
+| ---- | ---------- |
+| Integration | These are applications, either {{site.konnect_short_name}}-internal or external, that act as sources from which you can ingest resources. For example, GitHub. |
+| Resource | An umbrella term that denotes entities ingested by Service Catalog from enabled integrations. A resource can range from an infrastructural component (like Gateway services, mesh services, databases, and caches) to an external application or tool (like code repositories, CI/CD infrastructure, and on-call systems) to a piece of documentation (like API specs). Resources can be mapped to one or more Services. |
+| Service | A Service is a collection of resources in the Service Catalog. It is what a developer may regard as a "unit of software," typically a backend service exposing an API |
```

Comment:
> It is what a developer may regard as a "unit of software," typically a backend service exposing an API

@miko-bautista I copied this from the learning hub memo table you shared, but I wasn't sure about this part. I think this is what a Gateway service is, whereas the Service is a collection of resources that were injested and might not represent an API.

Reply:
Thanks for catching! Could we go with something like this instead?

"A unit of software that is typically owned by a single team, exposes one or more APIs, and may be dependent on other Services (as either upstream or downstream). In Service Catalog, a Service is represented as a collection of one or more Resources."